src/python/pants/option/global_options.py

```diff
@@ -901,14 +916,33 @@ def register_bootstrap_options(cls, register):
             type=list,
             default=DEFAULT_EXECUTION_OPTIONS.remote_store_addresses,
             help="host:port of grpc server to use as remote execution file store.",
+            removal_version="2.4.0.dev0",
+            removal_hint=(
+                "Use `--remote-store-address` instead.\n\nNote that you must add the scheme "
+                "prefix `grpc://` or `grpcs://`, i.e. gRPC with TLS enabled.\n\n"
```

Comment:
Maybe add "in order to identify the protocol to use" after "TLS enabled" ? Plus conform similar usages in other help messages.

Reply:
Hows the latest rewrite?